### PR TITLE
#21, 32: Replaced 'user' with user photo with dropdown menu

### PR DESCRIPTION
**Pull request by DavidScottBernstein**
#21, #32 User photo currently linked to user profile and no user menu
#31 Created drop down but no items have been implemented yet.

### CODE REVIEW

File: parts/nav-bftop.php [hunk 1]
```diff
@@ -11,8 +11,20 @@
 				<?php get_template_part( 'parts/nav', 'bfsubmenu' ); ?>
             </div>
         </div>
-        <div class="cell" style="width: 54px; height: 54px;"> 
-          User
+        <div class="cell" style="width: 54px; height: 54px;">
+        <a href="<?php bp_loggedin_user_link(); ?>" data-toggle="user-dropdown"><?php bp_loggedin_user_avatar( 'type=full' ); ?></a>
```
Comment (iangilman) [inline parts/nav-bftop.php]: One small detail: everything inside the div above should be indented one stop. That helps keep things organized.

File: parts/nav-bftop.php [hunk 2]
```diff
@@ -11,8 +11,20 @@
...
+        <div class="dropdown-pane" id="user-dropdown" data-dropdown data-hover="true" data-hover-pane="true" data-auto-focus="true">
+        <ul>
```
Comment (iangilman) [inline parts/nav-bftop.php]: Likewise here, this ul (and everything inside of it) should be indented from the div above it.

**Comment by iangilman** [comment]
Looks like @rgilman has found some of the links you'll need for the menu: 

```
<a href="/wp-admin/">Admin Dashboard</a>
<a href="/wp-login.php?action=logout">Logout</a>
<a href="/members/<?php bp_loggedin_user_username(); ?>/profile/edit/">Edit My Profile</a>
<a href="/members/<?php bp_loggedin_user_username(); ?>/profile/change-avatar/">Change Avatar</a>
<a href="/members/<?php bp_loggedin_user_username(); ?>/messages/">Messages</a>
<a href="/members/<?php bp_loggedin_user_username(); ?>/compose/">Compose Message</a>
<a href="/members/<?php bp_loggedin_user_username(); ?>/settings/">Email/Password Settings</a>
<a href="/members/<?php bp_loggedin_user_username(); ?>/settings/notifications/">Notification Settings</a>
<a href="/members/<?php bp_loggedin_user_username(); ?>/settings/profile/">Profile Settings</a>
```

**Comment by iangilman** [comment]
... And you might look at the documentation for the Foundation dropdown: 

https://foundation.zurb.com/sites/docs/dropdown-menu.html

Looks like you're doing it a little differently than that, but it's working just fine! I'm curious what the differences are.

**Comment by rgilman** [comment]
It looks to me that what @DavidScottBernstein has done with dropdown-pane is actually better in this case than my attempt with dropdown menu. Looks good to me for merging. I can then PR/merge the clean-up and recode work I've been doing.

**Comment by iangilman** [comment]
This is superseded by #36; closing.

**Pull request closed without merging**